react/features/toolbox/actionTypes.js

```diff
@@ -10,6 +10,19 @@ import { Symbol } from '../base/react';
 export const CLEAR_TOOLBOX_TIMEOUT = Symbol('CLEAR_TOOLBOX_TIMEOUT');
 
 /**
+ * The type of the action which sets default toolbar buttons.
+ *
+ * {
+ *      type: SET_DEFAULT_TOOLBAR_BUTTONS,
+ *      defaultToolbarButtons: {
```

Comment:
I really dislike actions that are deep object trees. You know eventually you will (or should be, at least) caring about the redux state that's `primaryToolbarButtons` and `secondaryToolbarButtons` so you cannot really accept a `defaultToolbarButtons` that contains only `makeUpThirdKindOfToolbarButtons`. Please try to avoid such superficial deep object trees for redux actions in the future.